release-notes/v3.28.0-release-notes.md

```diff
@@ -0,0 +1,120 @@
+03 May 2024
+
+#### GA support for IPv6 eBPF data plane
+
+Support for IPv6 and dual-stack when using Calico's eBPF data plane is now GA.
+
+#### Ensure pods are marked ready only after policies are rendered in dataplane
+
+The Calico CNI plugin can now optionally delay pod startup until after the local dataplane has been programmed. This may be desired in high churn scenarios to ensure network connectivity is avaialble from the moment the application software starts.
+
+#### Support for multiple IP pools in Tigera Operator installs
+
+The tigera operator now suppports configuring multiple IP pools at install time using the Installation API. IP pools can be modified after install via the Installation API as well.
+
+#### Bug fixes
+
+**Breaking change:** On upgrade, the UID of projectcalico.org/v3 resources will change. It is recommended that you restart any controllers that may care about this after upgrading Calico, including the kube-controller-manager. [calico #8586](https://github.com/projectcalico/calico/pull/8586) (@caseydavenport)
```

Comment:
Can this one go in an "Important Upgrade Notes" section at the top of the whole release notes page as #2 (see other comment for #1) please